azure-pipelines.yml

```diff
@@ -58,43 +58,13 @@ variables:
   GE_USAGE_STATISTICS_URL: "https://qa.stats.greatexpectations.io/great_expectations/v1/usage_statistics"
 
 stages:
-  - stage: scope_check
-    pool:
-      vmImage: 'ubuntu-20.04'
-    jobs:
-      - job: changes
-        steps:
-          - task: ChangedFiles@1
-            name: CheckChanges
-            inputs:
-              verbose: true
-              rules: |
-                [ContribChanged]
-                contrib/**
-
-                [ExperimentalChanged]
-                contrib/experimental/**
-
-                [DocsChanged]
-                docs/**
-                tests/integration/docusaurus/**
-                tests/integration/fixtures/**
-                tests/test_sets/**
-
-                [GEChanged]
-                great_expectations/**
-                tests/**
-                /*.txt
-                /*.yml
-
```

Comment:
Rather than these complex set of checks, we simply say if anything in the codebase has changed at all, run the full suite. While this is not as fine-tuned as `scope_check`, it seems to be the supported approach here. We'd rather err on the side of caution anyways.

If it seems like there are issues with this approach, I think we should change `always` to `true` and run tests every 6 hours.